src/fsstorage.h

```diff
@@ -41,8 +41,10 @@ class FSStorage : public INvStorage {
   virtual void clearEcuRegistered();
   virtual void storeInstalledVersions(const std::map<std::string, std::string>& installed_versions);
   virtual bool loadInstalledVersions(std::map<std::string, std::string>* installed_versions);
+  virtual StorageType type() { return type_; };
 
  private:
+  const StorageType type_;
```

Comment:
Could this be simplified to just:

~~~
virtual StorageType type() { return kFileSystem; };
~~~

Reply:
Oh, yes You are right